src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.hpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+
+#include "compiled_model.hpp"
+
+namespace ov {
+namespace npuw {
+
+class LLMCompiledModel : public ov::npuw::ICompiledModel {
+    using GetPropertiesMap =
+        std::map<std::string, std::tuple<ov::PropertyMutability, std::function<ov::Any(const ::intel_npu::Config&)>>>;
+public:
+    struct KVCacheDesc {
+        uint32_t max_prompt_size;
+        uint32_t total_size;
+        uint32_t num_stored_tokens;
+        uint32_t dim;
```

Comment:
let's put default values here. maybe `0`s

Reply:
Yes, thank you!

Reply:
Fixed